src/cpu.rs

```diff
@@ -173,21 +175,22 @@ pub(crate) mod arm {
         #[cfg_attr(
             any(
                 target_os = "ios",
+                target_os = "macos",
```

Comment:
A lot of my recent changes to cpu.rs were to clean up this repetitive logic that existed just to avoid dead code warnings.